File: Astronomy-ObjC/View Controllers/ImageCollectionViewCell.m
//
//  ImageCollectionViewCell.m
//  Astronomy-ObjC
//
//  Created by Chad Parker on 8/5/20.
//  Copyright © 2020 Chad Parker. All rights reserved.
//

#import "ImageCollectionViewCell.h"

@interface ImageCollectionViewCell ()

@property (strong, nonatomic) IBOutlet UIImageView *imageView;

@end

@implementation ImageCollectionViewCell


- (void)prepareForReuse
{
    [super prepareForReuse];
}

@end

File: Astronomy-objc/Astronomy-objc/Views/ImageCollectionViewCell.swift
//
//  ImageCollectionViewCell.swift
//  Astronomy-objc
//
//  Created by Bradley Yin on 10/15/19.
//  Copyright © 2019 bradleyyin. All rights reserved.
//

import UIKit

class ImageCollectionViewCell: UICollectionViewCell {
    
    @IBOutlet weak var imageView: UIImageView!
    
    override func prepareForReuse() {
        super.prepareForReuse()
        imageView.image = #imageLiteral(resourceName: "MarsPlaceholder")
    }
}
